src/towncrier/test/test_builder.py

```diff
@@ -132,3 +134,59 @@ def test_orphan_all_digits(self):
             parse_newfragment_basename("+123.feature", ["feature"]),
             ("+123", "feature", 0),
         )
+
+
+class TestIssueOrdering(TestCase):
```

Comment:
Can we have a docstring for this set of tests ?

I am not sure if this is for ordering for news fragments based on their issue number or for issue numbers inside a single fragment

Reply:
Both :D